### DISCUSSION TITLE
Want to open a new workbook with *.ipynb

### DISCUSSION THREAD

**Discussion by gauravchakrabarty** [cu-esiil-edu/2024-data-short-course#93] [Q&A]
@nquarder @eculler 
HI Mam/ Sir,
I want to open a blank workbook with the *.ipynb extension. 
I tried, but I am unable to open it.
I need to remove the instructions from the existing workbooks before the final submission. 
Please Help !!

**Comment by gauravchakrabarty** [comment]
> @nquarder @eculler HI Mam/ Sir, I want to open a blank workbook with the *.ipynb extension. I tried, but I am unable to open it. I need to remove the instructions from the existing workbooks before the final submission. Please Help !!

I followed the like opening a new file and saving in *.ipynb 

![pic](https://github.com/cu-esiil-edu/2024-data-short-course/assets/8917893/8c8d8d61-e949-4d71-a541-bd75f0654c5b)

Is it correct?

**Reply by nquarder** [reply]
That looks right @gauravchakrabarty ! 👍🏻

**Reply by gauravchakrabarty** [reply]
Thanks !!
